src/components/CardGrid.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react'
+import tw from 'twin.macro'
+import styled from 'styled-components'
+
+type Props = {
+  children: ReactNode
+}
+
+const CardGrid = ({ children }: Props) => {
+  return <Grid>{children}</Grid>
+}
+
+export default CardGrid
+
+const Grid = styled.div`
+  ${tw`grid grid-cols-1 px-40 justify-items-center`}
+
+  @media (min-width: 1000px) {
```

Comment:
definitely just use the built-in tailwind media-queries, they're soo much easier